lib/engine/minor.rb

```diff
@@ -57,5 +58,15 @@ def float!
     def inspect
       "<#{self.class.name}: #{id}>"
     end
+
+    def closed?
+      @closed
+    end
+
+    def close!(bank)
```

Comment:
just set owner nil

Reply:
I corrected that, but forgot to comment. So it has been fixed.